### PR TITLE
33 Fix windows compatibility issues on latest master branch

### PR DESCRIPTION
**Pull request by wuyingnan**

### CODE REVIEW

**Comment by wuyingnan** [comment]
This PR fixes #33

File: include/gobot/core/ref_counted.hpp [hunk 1]
```diff
@@ -20,7 +20,7 @@ using Ref = typename third_party::intrusive_ptr<T>;
 template <typename T>
 using RefWeak = third_party::intrusive_weak_ptr<T>;
 
-class RefCounted : public third_party::intrusive_base<RefCounted>, public Object  {
+class GOBOT_EXPORT RefCounted : public third_party::intrusive_base<RefCounted>, public Object  {
```
Comment (StephenYu-R) [inline include/gobot/core/ref_counted.hpp:23]: The third party intrusive_ptr has been dropped, need to confirm @Wuqiqi123

Comment (StephenYu-R) [reply include/gobot/core/ref_counted.hpp:23]: Postpone this issue to the next PR.

**Review by StephenYu-R — APPROVED** [review]
LGTM, approved.

**Pull request merged**